src/framework/fixture.ts

```diff
@@ -4,9 +4,10 @@ import { ParamsAny } from './params/index.js';
 // A Fixture is a class used to instantiate each test case at run time.
 // A new instance of the Fixture is created for every single test case
 // (i.e. every time the test function is run).
-export abstract class Fixture {
+export class Fixture {
   params: ParamsAny;
   protected rec: TestCaseRecorder;
+  private outstanding = 0;
```

Comment:
nit: Know this is a refactor, but maybe "outstandingRejections" or outstandingPromises?

Reply:
Good idea, now "numOutstandingAsyncExpectations"